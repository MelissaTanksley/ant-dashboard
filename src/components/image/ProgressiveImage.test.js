import React from 'react';
import { shallow } from 'enzyme';  
import ProgressiveImage from './progressiveImage';

describe('<ProgressiveImage />', () => {
    const ImageWrapper = shallow(ProgressiveImage);
    console.log(ImageWrapper)
    it('matches snapshot', () => {

    })
})