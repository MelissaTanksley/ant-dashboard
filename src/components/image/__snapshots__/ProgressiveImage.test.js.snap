// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProgressiveImage /> matches snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ProgressiveImage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "alt": undefined,
      "src": undefined,
      "style": Object {
        "filter": "blur(20px)",
        "height": "100%",
        "padding": "20px",
        "transition": "0.5s filter linear",
        "width": "100%",
      },
    },
    "ref": null,
    "rendered": null,
    "type": "img",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "alt": undefined,
        "src": undefined,
        "style": Object {
          "filter": "blur(20px)",
          "height": "100%",
          "padding": "20px",
          "transition": "0.5s filter linear",
          "width": "100%",
        },
      },
      "ref": null,
      "rendered": null,
      "type": "img",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
